### DISCUSSION TITLE
Change Domain

### DISCUSSION THREAD

**Discussion by RobertDWhite** [onedr0p/cluster-template#741] [Q&A] [answered]
Hello,

If one wants to change the domain used during install to another domain and set up a new cloudflared tunnel, will the entire process need to be re-run from scratch, or is there a recommended "go back here" step to easily change the domain?

Thanks for the great resource!

**Comment by onedr0p** [comment, marked as answer]
Could you change all references from your old domain to the new one and resync all the flux kustomizations and helmrelease?

**Reply by RobertDWhite** [reply]
I think this is correct. I got it to work by doing as you said, nuked the cluster, rebuilt the tunnel, and rebuilt the cluster. I think I was making it harder than it needed to be.
Thank you!
